src/loaders/ttf/tvgTtfLoader.h

```diff
@@ -37,12 +37,15 @@ struct TtfLoader : public FontLoader
     char* text = nullptr;
     Shape* shape = nullptr;
     bool italic = false;
+    bool fromMemory = false;
+    bool copy = false;
```

Comment:
this looks unnecessary.

Reply:
in tvgTtfLoader.cpp fun clear():
```
void TtfLoader::clear()
{
    if (fromMemory) {
        if (freeData) free(reader.data);
        reader.data = nullptr;
        reader.size = 0;
        freeData = false;
    } else _unmap(this);
    shape = nullptr;
}
```
@hermet  I wanted to avoid calling _unmap if _map wasn't called first. this is why I introduced fromMemory.

Reply:
@mgrudzinska 
1. Can you try to merge 2 vars?  `freeData` & `fromMemory`
2. Suggesting names: 
 - `direct`     //direct read from the data
 - `nomap`   //no mapping

Reply:
@mgrudzinska 

Could you please merge the 2 vars? `freeData`, `fromMemory` 
Suggested names:
 - `direct`      //direct read from the data
 - `nomap`    //no mapping

Reply:
I can introduce some enum, but with bool I don't see how to do this. and an enum doesn't seem like a good idea.
`if nomap` does not mean that I have to free the data

Reply:
> I can introduce some enum, but with bool I don't see how to do this.

it's ok. let me try it later.

> `if nomap` does not mean that I have to free the data

Yes, that's what I asked you to use `freeData` aligning with others. However, the condition says `mapping case` as you argued:

> I wanted to avoid calling _unmap if _map wasn't called first.

`nomap` is straightforward to use. `fromMemory` does not mean to either `free the data` or `map/unmap`.
```
if (nomap) { 
   //no-mapping scenario
} else {
   //mapping scenario
}
```